test/integration/default/serverspec/default_spec.rb

```diff
@@ -37,3 +37,7 @@
 describe command('/usr/local/bin/openstack -h') do
   its(:exit_status) { should eq 0 }
 end
+
+describe command("/opt/chef/embedded/bin/gem list -i -v '>= 0.2.0' fog-openstack") do
```

Comment:
Everything looks good, though is there any reason this isn't testing for the existence < 0.2.0 like the identity spec?

Reply:
@jcorme see my note above